### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Corey-Grainger**
@JarrodE the student you were schedueled to review this week has withdrawn.  They were intended to review my code this week so Lindsay has asked that I mention you instead. Could you please my code for week 6.  In particular I think my docstrings for my classes could be better. 

Thanks =)

### CODE REVIEW

File: prac_06/programming_language.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        """Programming language class"""
+        self.name = name
+        self.typing = typing
+        self.has_reflection = has_reflection
+        self.year = year
+
+    def __str__(self):
+        return f"{self.name}, {self.typing} Typing, Reflection={self.has_reflection}, First appeared in {self.year}"
+
+    def is_dynamic(self):
```
Comment (JarrodE) [inline prac_06/programming_language.py:17]: Missing a docstring?

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : ${self.cost:.2f})"
+
+    def get_age(self):
+        return 2023 - self.year
+
+    def is_vintage(self):
+        return (self.get_age()) >= 50
```
Comment (JarrodE) [inline prac_06/guitar.py]: Might be missing a couple more docstrings

**Review by JarrodE — COMMENTED** [review]
I wrote a couple of things to fix but as I continued to go next and then next again and again, it looks like you fixed everything I was looking at haha. Nothing else from what I can see with my amateur eyes. Good job! :)

**Comment by Corey-Grainger** [comment]
Cheers for the review 👍 .  Sorry about the changes, I had some stuff I fixed and obviously hadn't pushed to GitHub in a timely fashion :)

**Pull request merged**